### PR TITLE
Feature cucumber group

### PR DESCRIPTION
**Pull request by GustavoZHA**

### CODE REVIEW

File: src/test/java/salesforce/steps/groupGeneralSteps.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+import static utilities.JsonFormat.jsonConvert;
+
+public class groupGeneralSteps {
+    private Logger log = Logger.getLogger(getClass());
+    private ApiRequest apiRequest = new ApiRequest();
+    private ApiRequest apiRequestPatch = new ApiRequest();
+    private ApiResponse apiResponse;
+    private ObjectInformation objectInformation;
+
```
Comment (jpgonzales27) [inline src/test/java/salesforce/steps/groupGeneralSteps.java:39]: This line should be deleted

File: src/test/java/salesforce/steps/groupGeneralSteps.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    public groupGeneralSteps(ObjectInformation objectInformation) {
+        log.info("GroupGeneralSteps constructor");
+        this.objectInformation = objectInformation;
+    }
+
+
```
Comment (jpgonzales27) [inline src/test/java/salesforce/steps/groupGeneralSteps.java:46]: This line should be deleted

Comment (GustavoZHA) [reply src/test/java/salesforce/steps/groupGeneralSteps.java:46]: solved

File: src/test/java/salesforce/steps/groupGeneralSteps.java (hunk 1 above)
Comment (GustavoZHA) [reply src/test/java/salesforce/steps/groupGeneralSteps.java:39]: ok

**Comment by jpgonzales27** [comment]
Should be to add one line at the end of the file

**Pull request merged**